RevenueCatUI/CustomerCenter/Data/CustomerCenterAction.swift

```diff
@@ -0,0 +1,22 @@
+import RevenueCat
+
+/// Typealias for handler for Customer center actions
+public typealias CustomerCenterActionHandler = @MainActor @Sendable (CustomerCenterAction) -> Void
+
+/// Represents an event the customer may perform during the Customer Center flow
+public enum CustomerCenterAction {
+    /// Purchase of a promotional offer is completed.
+    case purchaseCompleted(_ customerInfo: CustomerInfo)
```

Comment:
should we be more explicit here about it being a promo offer? Maybe we can add data about the product that was purchased and the offer?

Reply:
Since this event is not being triggered, feel free to leave this one out of the PR if you want, I feel we might need to define events around feedback surveys and promo offers a bit better

Reply:
Yeah I think I will remove it for now 👍. As for it being a promo offer/data of what's purchased, I think we can add that as well once we add support for that event.